src/Plugin/Menu/CustomEditTab.php

```diff
@@ -0,0 +1,20 @@
+<?php
```

Comment:
@marlo-longley can you explain a little bit better why CustomEditTab class is needed? Just curious why we need to have extra logic to cast an Route Parameter, if that could be done via YAML directly

Reply:
@DiegoPino this seems to be the standard approach of how to pass route parameters in dynamic local tasks. I didn't see a way to do it directly in the tasks.yml. Without that piece I get errors.
https://www.drupal.org/docs/8/api/menu-api/providing-module-defined-local-tasks#s-customising-local-task-behavior